compiler/prelude/regen.go

```diff
@@ -0,0 +1,34 @@
+// +build ignore
+
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/gopherjs/gopherjs/compiler/prelude"
+)
+
+func main() {
+	cmd := exec.Command("uglifyjs")
+	cmd.Stdin = strings.NewReader(prelude.Prelude)
+
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatalf("uglifyjs: %v\n%s", err, out)
+		os.Exit(1)
+	}
+
+	err = ioutil.WriteFile("prelude_min.go", []byte(`// Code generated by regen. DO NOT EDIT.
+
+package prelude
+
+const Minified = `+"`"+string(out)+"`\n"), 0644)
```

Comment:
Is `out` guaranteed not to contain <code>`</code> characters?

If so, please document the reason. Otherwise, consider using `strconv.Quote` or something equivalent that is safe.

Reply:
Also, `Minified` should have a comment. It's an exported symbol.

Reply:
> Is `out` guaranteed not to contain ``` characters?

Good catch. I don't know, but I've now made it safe in the latest commit (we need to `strings.Replace`, not quote). In any case the worst that would have happened here is that we get a compile error. But that's a moot point because we now output the safe version.

> Also, `Minified` should have a comment. It's an exported symbol.

Done. And I've also commented `Prelude` because that wasn't commented.